src/views/memberHistory.vue

```diff
@@ -0,0 +1,96 @@
+<template>
```

Comment:
This is a very minor thing, but I think we should use UpperCamelCase for all of our .vue file names, just for consistency

Reply:
so instead of `memberHistory.vue`, the correct name would be `MemberHistory.vue`